pkg/config/integration_config.go

```diff
@@ -118,14 +119,28 @@ func buildLogsAgentIntegrationsConfig(config *viper.Viper, ddconfdPath string) e
 
 // availableIntegrationConfigs lists yaml files in ddconfdPath
 func availableIntegrationConfigs(ddconfdPath string) []string {
+	integrationConfigFiles := integrationConfigsFromDirectory(ddconfdPath, ".")
+	dirs, _ := ioutil.ReadDir(ddconfdPath)
+	for _, d := range dirs {
+		if d.IsDir() {
+			integrationConfigFiles = append(
+				integrationConfigFiles,
+				integrationConfigsFromDirectory(filepath.Join(ddconfdPath, d.Name()), d.Name())...,
```

Comment:
Would it be interesting to use a local variable to increase readability ?
```
path := filepath.Join(ddconfdPath, d.Name())
otherConfigs := integrationConfigsFromDirectory(path, d.Name())
integrationConfigFiles = append(integrationConfigFiles , otherConfigs...)
```

Reply:
this is relevant. Let's still merge given the timing, but do that next time we touch this part of the code